packages/EasyTest/src/Traits/MessengerAssertionsTrait.php

```diff
@@ -118,7 +118,7 @@ static function (object $message) use ($messageClass, $expectedProperties, $prop
     }
 
     /**
-     * @param array<int, array<class-string<\Throwable>, int|string>> $expectedExceptions
+     * @param array<int, class-string, array<class-string<\Throwable>, int|string>> $expectedExceptions
```

Comment:
It looks like a mistake. It should be:
`@param array<int, class-string<\Throwable>|array<class-string<\Throwable>, int|string>> $expectedExceptions`

Reply:
Fixed

Reply:
Looks the same.

Reply:
Thanks